src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/tests/ForeachStatement.ts

```diff
@@ -0,0 +1,49 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import { assertMatchesSnapshot } from './infrastructure/TestUtilities';
+
+// See GrammarTests.test.ts for details on exporting this test suite instead of running in place.
+
+export function RunForeachStatementSuite() {
+    describe('@foreach ( ... ) { ... }', () => {
```

Comment:
Just curious, how long does it generally take to run one of these tests?

Reply:
No more than 10 seconds for every grammar test that we have today 😄 . They're pretty darn fast